docs/08_troubleshooting.md

```diff
@@ -131,4 +149,16 @@ __Possible solution__: The key point here is the `expected order` and what you t
 }
 ```
 
-The code above has one `print` statement before the `singleton_set.send` and another one after the `singleton_set.send`. If you wrote some more `print` statements in the code that implements the `singleton_set.send` action and expect to see them before the second `print` statement then it is a wrong assumption. The inline actions are broadcasted to the network and they are executed at a different time, asynchronous of the current execution thread of the current `multi_index_example::mod` action, therefor it is impossible to predict when the `print` statements from inline action code will be outputted.
\ No newline at end of file
+The code above has one `print` statement before the `singleton_set.send` and another one after the `singleton_set.send`. If you wrote some more `print` statements in the code that implements the `singleton_set.send` action and expect to see them before the second `print` statement then it is a wrong assumption. The inline actions are broadcasted to the network and they are executed at a different time, asynchronous of the current execution thread of the current `multi_index_example::mod` action, therefor it is impossible to predict when the `print` statements from inline action code will be outputted.
+
+## Assertion failure while creating an account after eosio.system was installed
+
+```console
+cleos create account eosio bob EOS5HUanbay86UUnr1d4fuBsQ3ksjfgZYoLUVvrYVLy6pj4i8xqVY
+Error 3050003: eosio_assert_message assertion failure
+Error Details:
+assertion failure with message: system contract must first be initialized
+```
```

Comment:
Consider splitting the shell command and the output using \`\`\`sh and \`\`\`console, respectively.

Reply:
yes, that is the goal in general, this was a slip, thanks!